### PR TITLE
homework-1

### PR DESCRIPTION
**Pull request by Kaisar283**

### CODE REVIEW

File: src/main/java/Repository/ApartmentStorage.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package Repository;
```
Comment (Alexmansar) [inline src/main/java/Repository/ApartmentStorage.java]: in general - package names in uppercase
just repository 
the same for other packages

File: src/main/java/Repository/ApartmentStorage.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        apartments.sort(Comparator.comparing(
+                apartment -> apartment.getReservedBy() == null ? "" : apartment.getReservedBy().getName()
+        ));
+        return apartments.reversed();
+    }
+
+    public void sortedApartmentByReservationStatus(){
+        apartments.sort(Comparator.comparing(Apartment::isReserved).reversed());
+    }
+
```
Comment (Alexmansar) [inline src/main/java/Repository/ApartmentStorage.java]: I would returnb here list as well

File: src/main/java/console/UserConsole.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+import java.util.Scanner;
+
+public class UserConsole {
+    private final ApartmentService apartmentService = new ApartmentService();
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    public void start(){
+        System.out.println("Welcome to the Apartment Management System!");
+        System.out.println("Type 'help' for a list of commands.");
```
Comment (Alexmansar) [inline src/main/java/console/UserConsole.java]: you can use text bloks here 
https://www.baeldung.com/java-text-blocks

File: src/main/java/console/UserConsole.java [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+            double price = Double.parseDouble(commandParts[1]);
+            int id = apartmentService.registerApartment(price);
+            System.out.println("Apartment registered with price: " + price + ", apartment ID: " + id);
+        }catch (NumberFormatException e){
+            System.out.println("Invalid price format. Please enter a valid number.");
+        }
+    }
+
+    private void reserveApartment(String[] commandParts){
+        if (commandParts.length < 2){
```
Comment (Alexmansar) [inline src/main/java/console/UserConsole.java]: what is 2? create a constant and give names for such vars, please))

File: src/main/java/console/UserConsole.java [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+    private void listApartment(String[] commandParts){
+        int page = 1;
+        int pageSize = 5;
+
+        if (commandParts.length >= 2){
+            page = Integer.parseInt(commandParts[1]);
+        }
+        if(commandParts.length >= 3){
+            pageSize = Integer.parseInt(commandParts[2]);
```
Comment (Alexmansar) [inline src/main/java/console/UserConsole.java]: same here

File: src/main/java/console/UserConsole.java [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+        int pageSize = 5;
+        if (commandParts.length < 1){
+            System.out.println("Usage: sorted by <field> <page> <pageSize>");
+            return;
+        }
+
+        if(commandParts.length >= 4){
+            page = Integer.parseInt(commandParts[3]);
+        }
+        if(commandParts.length >= 5){
```
Comment (Alexmansar) [inline src/main/java/console/UserConsole.java]: and here :)

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Kaisar283 Hi)) I have same comments))please fix it)))

**Comment by Kaisar283** [comment]
@Alexmansar Hi.
Your review hepled improve my code, thanks.
I fixed your remaks, please check)

**Review by Alexmansar — APPROVED** [review]
@Kaisar283 you can merge it))

**Pull request merged**